### ISSUE TITLE
Alt-tab order broken for certain apps

### ISSUE THREAD

**Issue by mitchcapper** [FireCubeStudios/UnitedSets#46]
I have noticed some apps still show up independently in the taskbar when they are the active tab in US (and hide otherwise).  While a minor visual annoyance not the end of the world.   It seems this causes a secondary bug however in alt-tab recent order being broken.   One might expect if you hit alt-tab and release you are instantly in your previous window.  Hitting it again brings you to the window active when you first hit it.  No matter if you alt-tab to the US tab manually or to the window that sits in the taskbar the window order is now the same for alt-tab:
0 - The window that also shows in the taskbar
1 - US
2 - The window you were on prior to hitting alt-tab.

So to switch back you have to hit alt-tab-tab otherwise you just end up in the US window.

Clearly the taskbar visibility and this issue are related, but if we can't find a workaround for it appearing in the taskbar we may be able to at least detect this and remove ourselves from the tab order when active.

**Comment by Get0457** [comment]
Does this only happen to UWP apps? I remembered the last time I tested this only happen to UWP apps, but I might be wrong.

**Comment by mitchcapper** [comment]
No, certainly happened with WPF apps and non .net apps I believe.
